### ISSUE TITLE
Feature Request: Please add a Filter preview

### ISSUE THREAD

**Issue by Mannshoch** [FreshRSS/FreshRSS#2787]
I have a huge amount of RSS feeds I use in FreshRSS. To reduce the count of Article I use several text filters. To handle them better, please add a Preview mode.

The goal is to see which filter removes which article. So I'm able to detect wrongly removed ones better.

**Comment by Alkarex** [comment]
In the meantime, you should be able to try the filters in the search field to see their impact

**Comment by math-GH** [comment]
@Mannshoch is it still an Feature Request?

**Comment by Mannshoch** [comment]
Yes. It would helping with the regex if you get a feedback on your modification.
